pom.xml

```
@@ -63,7 +63,7 @@
     </pluginRepositories>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.361.4</jenkins.version>
+        <jenkins.version>2.410-rc33839.8e8205848340</jenkins.version>
```

https://github.com/jenkinsci/workflow-job-plugin/pull/352#discussion_r1226878951 by the way

---

Now out of date. To pick up https://github.com/jenkinsci/jenkins/pull/8110/commits/16408d859a4ef8a095cb96c6cbea51427f926d90 (and https://github.com/jenkinsci/jenkins/pull/8110/commits/c862d9bed1f5080b931396901df99a34a0419a0e, not that it should matter)
```suggestion
        <jenkins.version>2.415-rc33995.c862d9b_ed1f5</jenkins.version>
```